Solutions/SharpArch.Domain/SharpArch.Domain.csproj

```diff
@@ -10,11 +10,10 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SharpArch.Domain</RootNamespace>
     <AssemblyName>SharpArch.Domain</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>
-    </AssemblyOriginatorKeyFile>
+    <SignAssembly>false</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\SharpArch.snk</AssemblyOriginatorKeyFile>
```

Comment:
Wouldn't this cause assembly signing to kick in? we don't want to sign the assemblies.